cypher/cypher-docs/src/docs/dev/ql/start/index.asciidoc

```diff
@@ -34,9 +34,9 @@ Using the `START` clause explicitly in a query will cause the query to fall back
 | `db.index.explicit.auto.seekRelationships(key :: STRING?, value :: ANY?) :: (relationship :: RELATIONSHIP?)` | Get relationship from explicit automatic index. Replaces `START r=relationship:relationship_auto_index(key = 'A')`
 | `db.index.explicit.seekRelationships(indexName :: STRING?, key :: STRING?, value :: ANY?) :: (relationship :: RELATIONSHIP?)` | Get relationship from explicit index. Replaces `START r=relationship:relIndex(key = 'A')`
 | |
-| `db.index.explicit.searchRelationshipsIn(indexName :: STRING?, in :: NODE?, query :: ANY?) :: (relationship :: RELATIONSHIP?, weight :: FLOAT?)` | Search relationship from explicit index, starting at the node 'in'
-| `db.index.explicit.searchRelationshipsOut(indexName :: STRING?, out :: NODE?, query :: ANY?) :: (relationship :: RELATIONSHIP?, weight :: FLOAT?)` | Search relationship from explicit index, ending at the node 'out'
-| `db.index.explicit.searchRelationshipsBetween(indexName :: STRING?, in :: NODE?, out :: NODE?, query :: ANY?) :: (relationship :: RELATIONSHIP?, weight :: FLOAT?)` | Search relationship from explicit index, starting at the node 'in' and ending at 'out'
+| `db.index.explicit.searchRelationshipsIn(indexName :: STRING?, in :: NODE?, query :: ANY?) :: (relationship :: RELATIONSHIP?, weight :: FLOAT?)` | Search relationship in explicit index, starting at the node 'in'
```

Comment:
Perhaps you want to make a card to push these fixes back into the neo4j code?

Reply:
Already done: https://github.com/neo4j/neo4j/pull/10283